### ISSUE TITLE
Display data from grades & grade history

### ISSUE THREAD

**Issue by praneeth-bojanala5** [therealsujitk/android-vtop-chennai#23]
## Feature Request

Show grades and grade history

**Comment by therealsujitk** [comment]
@CrypticVader yes, the data was displayed for the old UI, I still haven't decided how to go about displaying the data in the current UI. Since grades are only available in the end of a semester, I decided to keep this for later.

**Comment by CrypticVader** [comment]
> Since grades are only available in the end of a semester, I decided to keep this for later.

Oh, that makes sense 😅

**Comment by praneeth-bojanala5** [comment]
> @CrypticVader yes, the data was displayed for the old UI, I still haven't decided how to go about displaying the data in the current UI. Since grades are only available in the end of a semester, I decided to keep this for later.

Yeah, but what about grade history?
